packages/fx-core/src/plugins/resource/bot/plugin.ts

```diff
@@ -65,7 +65,7 @@ export class TeamsBotImpl {
 
     return ResultFactory.Success(
       new QTreeNode({
-        type: NodeType.group,
+        type: "group",
```

Comment:
If no hurry, define it as constants or enums?

Reply:
Defining as constant can help to auto-complete when writing code: 
![image](https://user-images.githubusercontent.com/1658418/119770185-763af000-beee-11eb-9dae-758e50398e68.png)

Reply:
@jayzhang these strings are shared across all plugins, I think making them as variables and let plugins refer to the variables is better. Otherwise, once the strings changed, all plugins need change as well.

Reply:
We can have a discuss about this topics, seems different person has different options. Hope we can just keep this.

Reply:
Same question, is it possible to define enum or type instead of use string? It is easy for maintenance and reduce error